.github/workflows/integration-tests-against-emulator.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
I think after the workflow config has been updated, you can delete this script.

Reply:
Done.